extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -573,3 +573,24 @@ export async function showInfoMessageWithOutputChannel(message: string, outputCh
 	}
 }
 
+/**
+ * Recursively gets all the sql files at any depth in a folder
+ * @param folderPath
+ */
+export async function getSqlFilesInFolder(folderPath: string): Promise<string[]> {
+	// path needs to use forward slashes for glob to work
+	const escapedPath = glob.escapePath(folderPath.replace(/\\/g, '/'));
+	const sqlFilter = path.posix.join(escapedPath, '**', '*.sql');
```

Comment:
One of the issues I ran into was after the initial build, some sql files (pre/post deploy scripts) get added to the /obj folder. I had to exclude /obj/*.sql and /bin/*.sql from the build list. Can you verify that those don't get picked up, after a successful build with pre/post scripts?

Reply:
Can't new-style project files specify more complete globs than just that?  e.g. `Production*.sql`?  Do we need to pass the specific glob in?

Reply:
Removed `/obj/**/*/*.sql `and `/bin/**/*.sql` and also added support for other globbing patterns that are specified in the sqlproj